### PR TITLE
Add a spell replace function

### PR DESCRIPTION
**Pull request by kurkale6ka**
Proposed map: `map global user = '<a-i>w:spell-replace<ret>'`

### CODE REVIEW

**Comment by kurkale6ka** [comment]
It'd be good to have words red underlined btw (vs in red background)

**Comment by mawww** [comment]
We cant have underlining with another color from fg on terminals unfortunately.

**Comment by casimir** [comment]
What about using bold as emphasis rather than underline? Would it allow another color from fg?

**Comment by lenormf** [comment]
Aren't those things customizable already (faces) ?

File: rc/base/spell.kak [hunk 1]
```diff
@@ -54,3 +54,14 @@ Formats of language supported:
         } </dev/null >/dev/null 2>&1 &
     }
 }
+
+def spell_replace %{%sh{
```
Comment (mawww) [inline rc/base/spell.kak]: currently, commands use `-` as a word separator, not `_`

**Pull request merged**